src/land/smap9km_ssm2ioda.py

```diff
@@ -92,6 +93,7 @@ def _read(self):
         ecoli = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['EASE_column_index'][:].ravel()
         refsec = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['tb_time_seconds'][:].ravel()
 
+        deps = np.full_like(vals, 0.025)
```

Comment:
what is the repeating value of 0.025 a default value?  could this be set somewhere with a longer name?

Reply:
@BenjaminRuston, this is what Clara Drape @ClaraDraper-NOAA r asked me to do (see issue #1068.